### ISSUE TITLE
Color remapping

### ISSUE THREAD

**Issue by AmonShokhinAhmed** [lizrad/recursio#143]
Would be quite easy to implement, with colors being globally defined anyway.

**Comment by AmonShokhinAhmed** [comment]
Should be done when #172 is merged.
